app/test/dartdoc/golden/dygraph_0.1.1_index.out.html

```diff
@@ -1,8 +1,8 @@
 <html lang="en">
   <head>
     <meta name="robots" content="noindex"/>
-    <script async="async" src="https://www.googletagmanager.com/gtag/js?id=UA-26406144-13"/>
-    <script src="/static/js/gtag.js"/>
+    <script async="async" src="https://www.googletagmanager.com/gtm.js?id=GTM-MX6DBN9"/>
+    <script src="/static/js/gtm.js?hash=fltt8rc82ob3timp0tvnk6b136c2fqn0"/>
```

Comment:
@isoos we agree that it's better to have this hash here, or did we deliberately avoid this in dartdoc because it's generated?

Reply:
Come to think, we probably don't want this...

Reply:
I think this is better with the hash, as it allows better caching while it falls back to serve the content when needed.